### PR TITLE
macOS: add @2x label for retina resolutions

### PR DESCRIPTION
**Pull request by iandrewt**

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -1324,6 +1324,12 @@ getresolution() {
                     awk '/Resolution:/ {printf $2"x"$4" @ "$6"Hz, "}')"
             fi
 
+            scale_factor="$(/usr/libexec/PlistBuddy -c "Print DisplayAnyUserSets:0:0:Resolution" /Library/Preferences/com.apple.windowserver.plist)"
+
+            if [ "${scale_factor%.*}" == "1" ]; then
```
Comment (dylanaraps) [inline neofetch]:
```sh
[ "${scale_factor%.*}" == "1" ] && \
    resolution="${resolution// @/@2x @}"
```

**Comment by dylanaraps** [comment]
Other than the comment, LGTM :+1:

**Comment by iandrewt** [comment]
oops didn't read comment right, hold on

**Comment by dylanaraps** [comment]
LGTM :+1:

**Pull request merged**